src/argilla/client/sdk/workspaces/api.py

```diff
@@ -42,7 +40,121 @@ def list_workspaces(client: httpx.Client) -> Response[Union[List[WorkspaceModel]
     response = client.get(url=url)
 
     if response.status_code == 200:
-        parsed_response = [WorkspaceModel(**workspace) for workspace in response.json()]
+        parsed_response = [WorkspaceModel.construct(**workspace) for workspace in response.json()]
```

Comment:
By using the `construct` method, all automatic casting will be applied. For example `str`-> `UUID` or `str`-> `datetime` will be applied?

Reply:
No it wont', so maybe we just prefer to avoid `construct` usage? Even though it will just affect the `id` and the `inserted/created_at` attributes (and the last two ones are not to be modified, and a `str` formatted as an UUID can be parsed as UUID, so I see no issue. But if we want to be more strict with typing sure, we can just just avoid the `construct` to validate the fields (I was just using the latter when there were nested `pydantic.BaseModel`s)